Formula/cwb3.rb

```diff
@@ -0,0 +1,102 @@
+class Cwb3 < Formula
+  desc "Tools for managing and querying large text corpora with linguistic annotations"
+  homepage "https://cwb.sourceforge.io/"
+  license "GPL-2.0-or-later"
+
+  stable do
+    url "https://downloads.sourceforge.net/project/cwb/cwb/cwb-3.5-RC/cwb-3.4.33-src.tar.gz"
+    sha256 "947d5f6d710fd8f818c2dc027e991b898308d581cb9b89c4ce6db054fbb20948"
+  end
+
+  head do
+    url "svn://svn.code.sf.net/p/cwb/code/cwb/trunk"
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "pcre"
+  depends_on "readline"
+
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
+  uses_from_macos "ncurses"
+
+  def install
+    system("make", "all",
+      "PLATFORM=homebrew-formula", "SITE=homebrew-formula", "FULL_MESSAGES=1",
+      "PREFIX=#{prefix}", "HOMEBREW_ROOT=#{HOMEBREW_PREFIX}")
+    ENV.deparallelize
+    system("make", "install",
+      "PLATFORM=homebrew-formula", "SITE=homebrew-formula", "FULL_MESSAGES=1",
+      "PREFIX=#{prefix}", "HOMEBREW_ROOT=#{HOMEBREW_PREFIX}")
+  end
+
+  def post_install
+    default_registry = HOMEBREW_PREFIX/"share/cwb/registry"
+    default_registry.mkpath # make sure default registry exists
+  end
+
+  def caveats
+    default_registry = HOMEBREW_PREFIX/"share/cwb/registry"
+    <<~STOP
+      CWB default registry directory for this build: #{default_registry}
+    STOP
+  end
+
+  test do
+    resource("tutorial_data") do
+      url "https://cwb.sourceforge.io/files/encoding_tutorial_data.zip"
+      sha256 "bbd37514fdbdfd25133808afec6a11037fb28253e63446a9e548fb437cbdc6f0"
+    end
```

Comment:
This should be declared after the dependencies.

Reply:
I moved this into the test block so that it is only installed when needed for testing, but not (unnecessarily) for the compilation.  Fine to move, of course.